org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/compiler/JvmModelGenerator.xtend

```diff
@@ -839,12 +839,7 @@ class JvmModelGenerator implements IGenerator {
 	}
 
 	def getImportManager(ITreeAppendable appendable) {
```

Comment:
Should the XbaseCompiler work with TreeAppendable instead of ITreeAppendable on its APIs?

Reply:
@szarnekow from what I see and understand, the XbaseCompiler does not need anything more than what the `ITreeAppendable` interface provides and doesn't seem to make further assumptions.
On the contrary, `JvmModelGenerator` completely relies on `TreeAppendable`.

Reply:
Meh sadness. We cannot easily change signatures here. Casting seems the best we can do.